src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-multisite-sync-policy/rgw-multisite-sync-policy.component.ts

```diff
@@ -54,21 +89,105 @@ export class RgwMultisiteSyncPolicyComponent implements OnInit {
         flexGrow: 1
       }
     ];
+    const getSyncGroupName = () =>
+      this.selection.first() && `${encodeURIComponent(this.selection.first().groupName)}`;
+    const addAction: CdTableAction = {
+      permission: 'create',
+      icon: Icons.add,
+      routerLink: () => this.urlBuilder.getCreate(),
+      name: this.actionLabels.CREATE,
+      canBePrimary: (selection: CdTableSelection) => !selection.hasSelection
+    };
+    const editAction: CdTableAction = {
+      permission: 'update',
+      icon: Icons.edit,
+      routerLink: () => this.urlBuilder.getEdit(getSyncGroupName()),
+      name: this.actionLabels.EDIT
+    };
+    const deleteAction: CdTableAction = {
+      permission: 'delete',
+      icon: Icons.destroy,
+      click: () => this.deleteAction(),
+      disable: () => !this.selection.hasSelection,
+      name: this.actionLabels.DELETE,
+      canBePrimary: (selection: CdTableSelection) => selection.hasMultiSelection
+    };
+    this.tableActions = [addAction, editAction, deleteAction];
+    this.setTableRefreshTimeout();
```

Comment:
do we need this?

Reply:
I am not sure, it was there in other places, so kept it
WDYT?

Reply:
i don't think we need it. We do that in buckets page because of the performance issues.. or in other places where the API is not performant enough. and this prevents from auto-reload. And I don't think we need it here for now. If we identify any performance issues later on, probably then we can introduce it.